CONTRIBUTING.md

```diff
@@ -0,0 +1,147 @@
+# Contributing
+
+FV3core is actively developed by Vulcan, so please contact us if there is interest in making contributions in the near-term.
+Contributors names will be added to [AUTHORS.rst](https://github.com/VulcanClimateModeling/fv3core/blob/master/AUTHORS.rst).
+
```

Comment:
The README does start with a 'Getting Started' section, not sure how much should be different for a Quickstart?

Reply:
I added something, not sure if it's what you had in mind?

Reply:
I think it's a great start, and good enough to merge. I'd like to modify it in another PR afterwards. It would be good for it to describe commands needed to install pre-commit and its dependencies, and I have some ideas to make it more terse.

Reply:
I guess it's a matter of whether it's a quickstart for developers or users of the code. currently developers make the most sense because we don't have users, though generally I think of a Quickstart as being a quick way for someone to run the code, and developers should spend the time to read more...

Reply:
I'd be fine with putting it in CONTRIBUTING instead, if that's more appropriate.

Given the amount of content in the readme, I hope the Quickstart can make 100% sure that people developing the code at least can run the tests (linting and pytest), even if they decide not to read the rest of the document. Of course they should read more, but at least this core functionality should be easy to find and reference.

We're only close to having a supported way for a "user" to run the code, the mini-app is still being written.